spec/features/admin/admin_modifies_items_spec.rb

```diff
@@ -31,7 +31,7 @@
     expect(page).to have_content "It Came From Beneath the Sink"
   end
 
-  xscenario 'can view item details' do
+  scenario 'can view item details' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.